query10.sql

```diff
@@ -4,4 +4,5 @@
 */
 
 -- Enter your SQL query here
-select ...
+select name, st_Distance(the_geom,st_setsrid(st_MakePoint(39.9522, -75.1927),4326), false) as distance_from_Meyerson
```

Comment:
I actually didn't know that by simply specifying `false` for the 3rd parameter that PostGIS would convert the `geometry` points to `geography`!